frontend/src/features/homes/home-group.vue

```diff
@@ -0,0 +1,190 @@
+              <q-icon class="legend-active" name="fa fa-square" />&nbsp;
+              {{ $i18n.t("homeGroup-activityLevelsLegend-active") }}
+            </div>
+          </q-th>
+        </template>
+
+        <template v-slot:body-cell-viewHome="props">
+          <q-td :props="props">
+            <q-btn size="sm" outline class="text-black bg-white">
+              <q-icon name="fa fa-eye" class="q-mr-sm" />
+              {{ $i18n.t("homeGroup-home-viewButton-text") }}
+            </q-btn>
+          </q-td>
+        </template>
+
+        <template v-slot:body-cell-viewReport="props">
+          <q-td :props="props">
+            <q-btn size="sm" outline class="text-black bg-white">
+              <q-icon name="fa fa-chart-line" class="q-mr-sm" />
+              {{ $i18n.t("homeGroup-home-reportButton-text") }}
+            </q-btn>
+          </q-td>
+        </template>
+      </q-table>
+    </q-card-section>
+
+    <add-home v-if="showAddHome" :group-id="group._id" @close="close" />
+    <assign-manager
+      v-if="showAssignManager"
+      :group-id="group._id"
+      @close="closeManager"
+    />
+  </q-card>
+</template>
+
+<script>
+import { getHomesWithActivityLevel } from "./services/homes-list";
+import { renderChart } from "./services/home-activity-levels.chart";
+import AddHome from "./add-home";
+import AssignManager from "./assign-manager";
+
+export default {
+  props: {
+    group: { type: Object, required: true }
+  },
+
+  components: {
+    AddHome,
+    AssignManager
+  },
+
+  data() {
+    return {
+      homes: [],
+      loading: false,
+      showAddHome: false,
+      showAssignManager: false,
+      columns: [
+        {
+          name: "viewHome",
+          align: "left",
+          field: "_id",
+          label: this.$i18n.t("homeGroup-tableHeader-viewHome")
+        },
+        {
+          name: "name",
+          align: "left",
+          field: "name",
+          label: this.$i18n.t("homeGroup-tableHeader-name")
+        },
+        {
+          name: "activityLevel",
+          align: "left",
+          style: "width: 600px",
+          field: "activityLevel",
+          label: this.$i18n.t("homeGroup-tableHeader-activityLevels")
+        },
+        {
+          name: "viewReport",
+          align: "left",
+          field: "_id",
+          label: this.$i18n.t("homeGroup-tableHeader-viewReport")
+        }
+      ]
+    };
+  },
+
+  async mounted() {
+    await this.init();
+  },
+
+  methods: {
+    async close(results) {
+      this.showAddHome = false;
+      if (results) {
+        await this.init();
+      }
+    },
+    closeManager() {
+      this.showAssignManager = false;
+    },
+    async init() {
+      this.homes = await getHomesWithActivityLevel(this.group._id);
+      await this.$nextTick();
+      this.homes.forEach(home => renderChart(home._id, home.activityLevel));
+    }
+  }
+};
+</script>
+<style lang="scss" scoped>
+.legend-inactive {
+  color: #d9534f;
```

Comment:
Can we use SCSS variables here, so that our colors are consistent throughout the app? E.g. variables for 
- inactive-color
- semiactive-color
- active-color

Reply:
done